package.json

```diff
@@ -4,7 +4,9 @@
   "description": "Blue Ocean acceptance tests",
   "main": "index.js",
   "scripts": {
-    "test": "nightwatch"
+    "test": "npm run nightwatch",
+    "nightwatch": "PATH=./node:./node/npm:$PATH nightwatch",
```

Comment:
Add to the path:

./node_modules/.bin

So we are sure nightwatch it's taken from there